tests/Smoke.hs

```diff
@@ -172,8 +172,7 @@ testContention (newChan,writeChan,readChan,_) writers readers n = do
   mapM_ (forkCatching False "testContention writeChan i " . mapM_ (writeChan i)) groups
 
   ns <- replicateM nNice (C.readChan out)
-  isEmpty <- C.isEmptyChan out
-  if sort ns == [1..nNice] && isEmpty
```

Comment:
I had to remove `isEmpty` because it is no long in base. Does this mess up the test?